modules/gdscript/gdscript.cpp

```diff
@@ -126,7 +126,10 @@ GDScriptInstance *GDScript::_create_instance(const Variant **p_args, int p_argco
 		GDScriptLanguage::singleton->lock->unlock();
 #endif
 
-		ERR_FAIL_COND_V(r_error.error != Variant::CallError::CALL_OK, NULL); //error constructing
+		if (r_error.error != Variant::CallError::CALL_OK) {
+			memdelete(instance);
```

Comment:
This one causes a crash, the instance is actually deleted earlier (when it does `instance->owner->set_script_instance(NULL);`). I'll remove it together with other GDScript fixes.